ledger-service/http-json/src/main/scala/com/digitalasset/http/Main.scala

```diff
@@ -49,12 +49,14 @@ object Main extends StrictLogging {
         s", portFile=${config.portFile: Option[Path]}" +
         s", applicationId=${config.applicationId.unwrap: String}" +
         s", packageReloadInterval=${config.packageReloadInterval.toString}" +
+        s", packageMaxInboundMessageSize=${config.packageMaxInboundMessageSize.toString}" +
```

Comment:
```suggestion
        s", packageMaxInboundMessageSize=${config.packageMaxInboundMessageSize: Option[Int]}" +
```

`toString` is applied automatically, so to speak.